Security/TwoFactor/AuthenticationContextInterface.php

```diff
@@ -63,4 +63,11 @@ public function isAuthenticated();
      * @param bool $authenticated
      */
     public function setAuthenticated($authenticated);
+
+    /**
+     * Is authentication request.
+     *
+     * @return bool
+     */
+    public function isAuthenticationTry();
```

Comment:
This is a backwards compatibility break. Adding a new method to AuthenticationContextInterface means, that everyone who implemented their own AuthenticationContext class based on that interface, would need to update the implementation.